addons/website/static/src/js/content/website_root.js

```diff
@@ -43,6 +43,7 @@ var WebsiteRoot = BodyManager.extend({
     }),
     custom_events: _.extend({}, BodyManager.prototype.custom_events || {}, {
         animation_start_demand: '_onAnimationStartDemand',
+        animation_stop_demand: '_onAnimationStopDemand',
```

Comment:
@qsm-odoo Diff needed for 11.0 (only change made since your review for 12.0)

Reply:
Mmh yes, I introduced animation stop on editor save in 12.0, before that all was handled manually in options' cleanForSave methods... Guess it is still fine to merge this in 11.0 anyway :)